public/src/client/chats.js

```diff
@@ -530,6 +532,72 @@ define('forum/chats', ['components', 'string', 'sounds', 'forum/infinitescroll',
 			callback();
 		});
 	}
+	
+	Chats.handleSearch = function() {
+		var timeoutId = 0;
+		
+		components.get('chat/search').on('keyup', function() {
+			if (timeoutId) {
+				clearTimeout(timeoutId);
+				timeoutId = 0;
+			}
 
+			timeoutId = setTimeout(doSearch, 250);	
+		});
+		
+		function doSearch() {
+            		var username = components.get('chat/search').val();
+            		var chatsListEl = $('[component="chat/search/list"]');
+            
+            		if(username) {
+	            		socket.emit('user.search', {
+	    			query: username,
+	    			searchBy: 'username'
+	        		}, function(err, data) {
+	        			if (err) {
+	        				return app.alertError(err.message);
+	        			}
+	                    
+	                    		chatsListEl.empty();
+	                    
+	                    		if (data.users.length === 0) {
+	                    			chatsListEl.append('<li><div><span>No users found!</span></div></li>');
```

Comment:
You can change this to 

`chatListEl.translateHtml('<li><div><span>[[users:no-users-found]]</span></div></li>');`

So it can use the language strings.